packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -51,24 +51,45 @@ enum DragStartBehavior {
 /// Configuration of multi-finger drag strategy on multi-touch devices.
 ///
 /// When dragging with only one finger, there's no difference in behavior
-/// between the two settings.
+/// between all the settings.
 ///
 /// Used by [DragGestureRecognizer.multitouchDragStrategy].
 enum MultitouchDragStrategy {
   /// Only the latest active pointer is tracked by the recognizer.
   ///
-  /// If the tracked pointer is released, the latest of the remaining active
+  /// If the tracked pointer is released, the first accepted of the remaining active
   /// pointers will continue to be tracked.
   ///
   /// This is the behavior typically seen on Android.
   latestPointer,
 
+  /// All active pointers will be tracked together. The scrolling offset
+  /// is determined by the maximum deltas of both directions.
+  ///
+  /// If the user is dragging with 3 pointers at the same time, each having
+  /// \[+10, +20, +33\] pixels of offset, the recognizer will report a delta of 33 pixels.
+  ///
+  /// If the user is dragging with 5 pointers at the same time, each having
+  /// \[+10, +20, +33, -1, -12\] pixels of offset, the recognizer will report a
+  /// delta of (+33) + (-12) = 21 pixels.
+  ///
```

Comment:
Mention the panning algorithm here, and change the first paragraph.

And I feel like the name `maxAllPointers` doesn't apply to panning, but I can't think of a better name either. 

Maybe `marginPointers`, in contrast to `latestPointer`, to indicate that the result comes from all the outermost ones. For one-dimensional movement, they're positive-most one and the negavie-most one. For two-dimensional movement, they're essentially all pointers. 

This is my suggestion of the entire documentation:
```dart
  /// All active pointers will be tracked, and the result is computed from the "outermost" pointers.
  ///
  /// The scrolling offset is determined by the maximum deltas of both directions.
  ///
  /// If the user is dragging with 3 pointers at the same time, each having
  /// \[+10, +20, +33\] pixels of offset, the recognizer will report a delta of 33 pixels.
  ///
  /// If the user is dragging with 5 pointers at the same time, each having
  /// \[+10, +20, +33, -1, -12\] pixels of offset, the recognizer will report a
  /// delta of (+33) + (-12) = 21 pixels.
  ///
  /// The panning offset is the average of all pointers.
  ///
  /// If the user is dragging ... [insert an example here].
  ///
  /// This is the behavior typically seen on iOS.
```

Reply:
We might need a native speaker for the final verdict. @Piinks May we have your idea? Does the doc and the variable name look clear enough to you?

Reply:
I also provide two names for reference, `mergePointers` or `conbinePointers`.

Reply:
I suggest against the word `sum`, because it's not a summation. The rule `sumAllPointers` literally adds all deltas together, which leads to twice the scrolling delta for two pointers. Other than that, I think either `extremePointers` or `boundaryPointers` sound ok. (`directionalMaxPointers` sounds a little too lengthy for me for no extra benefits.)

Reply:
If we want a verb, maybe we can use `averageBoundaryPointers`, which applies to both the one-directial algorithm and the panning algorithm.